Makefile

```diff
@@ -207,7 +207,7 @@ localunit: test/goecho/goecho varlink_generate
 	./contrib/cirrus/lib.sh.t
 
 ginkgo:
-	ginkgo -v -tags "$(BUILDTAGS)" $(GINKGOTIMEOUT) -cover -flakeAttempts 3 -progress -trace -noColor -nodes 3 test/e2e/.
+	ginkgo -v -tags "$(BUILDTAGS)" $(GINKGOTIMEOUT) -cover -flakeAttempts 3 -progress -trace -noColor -nodes 3 -debug test/e2e/.
```

Comment:
@vrothberg this option is needed with `-nodes 3` to make ginkgo log output from each node.  Otherwise, if any node hangs for any reason, it's incredibly difficult to debug which test caused the problem.

Reply:
Initially we just wanted it for debug, so I added the quick "HACK" message. Changed now.